x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -441,15 +509,24 @@ export class EndpointDocGenerator {
    * @param entityID - entityID of the originating process
    * @param parentEntityID - optional entityID of the parent process, if it exists
    * @param ancestry - an array of ancestors for the generated alert
+   * @param alertsDataStream the values to populate the data_stream fields when generating alert documents
    */
-  public generateAlert(
+  public generateAlert({
```

Comment:
Switching this to take a destructured object since it has quite a few parameters.